Code/Framework/AzFramework/AzFramework/Asset/AssetCatalog.h

```diff
@@ -98,8 +98,8 @@ namespace AzFramework
 
         //////////////////////////////////////////////////////////////////////////
         // NetworkAssetUpdateInterface
-        void AssetChanged(AzFramework::AssetSystem::AssetNotificationMessage message) override;
-        void AssetRemoved(AzFramework::AssetSystem::AssetNotificationMessage message) override;
+        void AssetChanged(const AZStd::vector<AzFramework::AssetSystem::AssetNotificationMessage>& messages, bool isCatalogInitialize = false) override;
```

Comment:
This are public APIs, did we consider deprecation as a strategy here? This will break legacy code integrated with these notifications. 

Or are these notifications still internal?

Reply:
Its the same internal API:

```
//! This internal interface exists only to communicate to an Asset Catalog or manager of Asset Catalogs
//! such as the PlatformAddressedAssetCatlogManager
//! If you want to know when new assets are ready to load or have changed, listen to the AssetCatalogEventBus
//! which will notify you when assets have been updated on the main thread only, and only after the catalog itself has
//! updated with the new information.
class NetworkAssetUpdateInterface
```